1.5/Source/PawnStorages/PawnStorages/Farm/Comps/CompFarmNutrition.cs

```diff
@@ -55,6 +55,7 @@ public override void PostExposeData()
         {
             base.PostExposeData();
             Scribe_Values.Look(ref _storedNutrition, "storedNutrition");
+            Scribe_References.Look(ref AlternativeStore, "AlternativeStore");
```

Comment:
This also fixes a bug I found where nutrition in pipe based systems wasn't being persisted.

Reply:
awesome spot.